lib/fluent/plugin/in_tail.rb

```diff
@@ -369,8 +369,10 @@ def existence_path
     def refresh_watchers
       target_paths_hash = expand_paths
       existence_paths_hash = existence_path
-
-      log.debug { "tailing paths: target = #{target_paths.join(",")} | existing = #{existence_paths.join(",")}" }
+      
+      target_paths_str = target_paths_hash.collect { |key, target_info| target_info.path }.join(",")
+      existence_paths_str = existence_paths_hash.collect { |key, target_info| target_info.path }.join(",")
```

Comment:
The debug string should be built only when debug mode is enabled. I'll add a commit to fix it.

Reply:
Done.